utils.bal

```diff
@@ -20,11 +20,12 @@ import ballerina/encoding;
 import ballerina/http;
 import ballerina/lang.'array as arrays;
```

Comment:
no need to put `'` before array because array is not a keyword.
We can import it like 
```
import ballerina/lang.array;
```

and use the functions like

```
array:toBase16()
```